src/Sylius/Behat/Page/Admin/Country/UpdatePage.php

```diff
@@ -49,6 +51,19 @@ public function disable()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function isCodeFieldDisabled()
+    {
+        try {
+            $codeField = $this->getElement('code');
+            return $codeField->getAttribute('disabled') === 'disabled';
+        } catch (ElementNotFoundException $exception) {
+            return false;
```

Comment:
Method name indicate that it should return boolean value so IMO we shouldn't throw exception.